### PR TITLE
Feature/table view

### PR DESCRIPTION
**Pull request by 0xLeif**
Here is an example project: https://github.com/0xLeif/Test-SUIK-TableView

TODO

- [x] Get Feedback
- [x] Implement more UITableView Delegate and DataSource methods
- [x] Support Cell Header and Footer
~~- [ ] Support Table Header and Footer~~
- [x] Write Tests

### CODE REVIEW

**Comment by 0xLeif** [comment]
Closes: https://github.com/0xLeif/SwiftUIKit/issues/88

**Comment by 0xLeif** [comment]
🧪 Test web hook 📬

File: Sources/SwiftUIKit/Views/TableView.swift [hunk 1]
```diff
@@ -0,0 +1,201 @@
+//
+//  TableView.swift
+//  SwiftUIKit
+//
+//  Created by Zach Eriksen on 4/12/20.
+//
+
+import UIKit
+
+@available(iOS 9.0, *)
```
Comment (ambid17) [inline Sources/SwiftUIKit/Views/TableView.swift:10]: is there a way to say all of the functions in this file have this availability? seem repetitive

Comment (0xLeif) [reply Sources/SwiftUIKit/Views/TableView.swift:10]: No... They would need to be under a class and then that class could be marked as available in iOS >9 ... But it doesn't work for the setup I have.

**Pull request merged**